content/src/content/jcr_root/apps/core/wcm/components/page/v2/page/README.md

```diff
@@ -19,10 +19,10 @@ Extensible page component written in HTL.
 
 ## Features
 * Editable templates
-* Page title, subtitle, description and thumbnail
+* Page title, subtitle, description, and thumbnail
```

Comment:
this new style, if introduced, should be applied to all lists. As it works fine without, I would prefer the original style of no serial comma.